setup.py

```diff
@@ -196,29 +227,32 @@ def run(self):
         # It's an old-style class in Python 2.7...
         distutils.command.clean.clean.run(self)
 
+def build_ext_with_dependencies(self):
+    build_dependencies()
+    return BuildExtension.with_options(no_python_abi_suffix=True)(self)
```

Comment:
We recently landed Ninja support which spawns a lot of threads due to parallel build. The CI might fail until we introduced a flag to control the number of threads. You can disable ninja by explicitly passing "use_ninja=False" to with_options.

Reply:
Might explain some failures of the CI. Thanks! 
I'll try it.

Reply:
Thanks for the tip, It helped with the linux_conda based pipeline. Tests are still failling, but not because of timeouts.
I'll keep investigating.